server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/devCountAdvertisersFunctions.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Description: Cloud functions for DEV count advertisers.git s      
+ * Author: Robert Marcus
+ * Date: July 14, 2020
+ */
+
+// Import links to firebase dev databases.
+const { DEV_ADS_DOCS } = require('../firebaseConfig');
+
+// Import links to firebase dev advertisers collections.
+// These are used to write directly to a collection.
+const { DEV_AGGREGATES_COLLECTION } = require('./countAdvertisersConfig');
+
+// Import helper functions.
+const { incrementAdvertiserCount, decrementAdvertiserCount } = 
+    require('./countAdvertisersHelpers');
+
+/**
+ * Increments an advertisers annual aggregate by one for an ad in a given year
```

Comment:
You miss a lot of apostrophes w/ the word "advertisers". I know the grammar of the comments isn't the most "important" part of the PR, but I do think it's worth the fix.